src/pcapi/core/offerers/models.py

```diff
@@ -229,6 +239,10 @@ def timezone(cls):  # pylint: disable=no-self-argument
             else_=case(CUSTOM_TIMEZONES, value=cls.departementCode, else_=METROPOLE_TIMEZONE),
         )
 
+    def field_exists_and_has_changed(self, field, value):
+        field_is_column = field in type(self).__table__.columns
+        return field_is_column and getattr(self, field) != value
```

Comment:
Y a-t-il des cas où l'on veut ignorer, de façon transparente, des arguments incorrects passés à `update_venue()`? J'ai envie de dire non : si du code appelle la fonction `update_venue()` avec des paramètres incorrects, j'aurais plutôt envie que ça lève une erreur.

Suggestion :

```suggestion
        if field not in type(self).__table__.columns:
            raise ValueError(f"Unknown field {field} for model {type(self)}")
        return getattr(self, field) != value
```